app/models/group.rb

```diff
@@ -1,30 +1,37 @@
+# == Schema Information
+#
+# Table name: groups
+#
+#  id                       :bigint           not null, primary key
+#  name                     :text             not null
+#  grp_type                 :string(20)
```

Comment:
Can you remove the grp_ prefix?

Reply:
This is the old type here, some PRs ago I created a `grp_type` / string -> `type` / enum migration. This should be removed later on, but out of the scope for this PR